bdns/dns.go

```diff
@@ -114,6 +115,73 @@ var (
 			Mask: []byte{255, 192, 0, 0},
 		},
 	}
+	privateV6Networks = []net.IPNet{
```

Comment:
You are missing a few blocks, they may reside in some of these existing ones but I'd really prefer we note them all here even so as it makes reading which blocks are actually being excluded and for what reason much easier to figure out.

Namely:
* 2001:2::/48 -- Benchmarking RFC5180
* 2001:db8::/32 -- Documentation RFC3849
* 2001::/32 -- TEREDO RFC 4380